test/v1/connectors/DeleteOtherDeductionsConnectorSpec.scala

```diff
@@ -16,45 +16,61 @@
 
 package v1.connectors
 
-import mocks.MockAppConfig
+import api.models.domain.TaxYear
 import v1.models.domain.Nino
-import v1.mocks.MockHttpClient
 import v1.models.outcomes.ResponseWrapper
 import v1.models.request.deleteOtherDeductions.DeleteOtherDeductionsRequest
 
 import scala.concurrent.Future
 
 class DeleteOtherDeductionsConnectorSpec extends ConnectorSpec {
-  val taxYear = "2017-18"
-  val nino    = "AA123456A"
-
-  class Test extends MockHttpClient with MockAppConfig {
-    val connector: DeleteOtherDeductionsConnector = new DeleteOtherDeductionsConnector(http = mockHttpClient, appConfig = mockAppConfig)
-    MockAppConfig.ifsBaseUrl returns baseUrl
-    MockAppConfig.ifsToken returns "ifs-token"
-    MockAppConfig.ifsEnvironment returns "ifs-environment"
-    MockAppConfig.ifsEnvironmentHeaders returns Some(allowedIfsHeaders)
-  }
 
-  "delete" must {
-    val request = DeleteOtherDeductionsRequest(Nino(nino), taxYear)
+  "delete" should {
```

Comment:
small thing, but may we amend this to read "deleteOtherDeductionsConnector"

Reply:
Updated, thanks.